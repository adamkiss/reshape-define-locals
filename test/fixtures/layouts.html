<define-locals>
	page:
		title: 'This is merged title'
	list: [merged 1, merged 2]
	deep:
		merge:
			object:
				key: This will be overwritten
</define-locals>
<extends src="layouts-base.html">
<block name="title"><title>{{ page.title }}</title></block>
<block name="content"><define-locals>
deep:
	merge:
		object:
			key: value
</define-locals><ul id="very-amaze"><each loop="item in list"><li>{{ item }}</li></each></ul><include src="layouts-include.html" /></block>
</extends>