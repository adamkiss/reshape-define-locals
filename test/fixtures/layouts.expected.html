<html>
<head><title>This is merged title</title></head>
<body><div id="content"><ul id="very-amaze"><li>original 1</li><li>original 2</li><li>merged 1</li><li>merged 2</li></ul><div id="included">The value of deep.merge.object.key is value</div></div></body>
</html>